tileserver/__init__.py

```diff
@@ -433,7 +434,13 @@ def read_tile(self, coord, offset=None):
         else:
             fmt = json_format
 
-        raw_data = self.store.read_tile(coord, fmt, 'all')
+        try:
+            raw_data = self.store.read_tile(coord, fmt, 'all')
+        except BotoServerError:
+            stacktrace = format_stacktrace_one_line()
+            print 'Error reading coord %s with format %s: %s' % (
+                serialize_coord(coord), format.extension, stacktrace)
```

Comment:
Does this need a `return None` after it?

I tried a little test to see if `raw_data` would be `None` following the `try` block, but it failed with `NameError: name 'raw_data' is not defined`.